data/json/items/melee.json

```diff
@@ -961,7 +961,7 @@
         "type":"GENERIC",
         "id" : "javelin",
         "name" : "wooden javelin",
-        "description" : "A wooden spear, honed to a sharper point and fire hardened for toughness.  The grip area has also be carved and covered for better grip.",
+        "description" : "A wooden spear, honed to a sharper point and fire hardened for toughness.  The grip area has been carved and covered for better grip.",
```

Comment:
While you're at it:

"A wooden spear honed to a sharper point and fire-hardened for toughness"

Comma isn't needed and "fire-hardened" [is preferable](https://github.com/atlaua/Cataclysm-DDA/blob/8951a491ca8133abba2f2970bc5cd97adcc2f82a/data/json/items/melee.json#L593).